atat_provisioning_wizard_api.yaml

```
@@ -441,6 +481,9 @@ components:
         properties:
           task_order_number:
             type: string
+          task_order_file_id:
+            type: string
+            description: ID of the file associated with this task order (previously uploaded by POST /taskOrderFiles)
```

In think the word _previously_ could be misleading.

```suggestion
            description: ID of the file associated with this task order (uploaded via POST /taskOrderFiles)
```